chapter_12_organising_test_files.asciidoc

```diff
@@ -517,8 +517,6 @@ and we'll adapt it slightly:
 ----
 ====
 
-// DAVID: I have `raise e` from that other method, not `raise`.
```

Comment:
I checked in Chapter 6 and it should be just `raise`. We hadn't dealt with the raised error object earlier and after having quickly checked I believe the first time we will is in Chapter 20.